src/content/en/tools/lighthouse/_toc.yaml

```diff
@@ -46,6 +46,8 @@ toc:
   path: /web/tools/lighthouse/audits/critical-request-chains
 - title: Displays Images With Incorrect Aspect Ratio
   path: /web/tools/lighthouse/audits/aspect-ratio
+- title: Document Does Not Have A Meta Description
```

Comment:
Can this be rephrased affirmatively, eg "Document Has A Meta Description"? (here and throughout)

Reply:
I'm using the negative version of each, because that is when readers are most likely to go to a doc.

Fail audit > reader sees "document does not have a meta description" in LH report > they go to the doc > doc title matches what they saw in their failing report

If we use the affirmative, then the doc title is inconsistent with what they saw in their report

Reply:
Ok makes sense. Do you have a plan to reword some of the other pages that are still affirmative?

Reply:
Yes